### DISCUSSION TITLE
bCNC for grbl5X?

### DISCUSSION THREAD

**Discussion by FKLCo** [vlachoudis/bCNC#1668] [Q&A]
Hallo sir
Is bCNC compatibile with grbl5X?, I try but without succes to connect.

**Comment by MARIOBASZ** [comment]
Yes

**Reply by FKLCo** [reply]
![bcnc](https://user-images.githubusercontent.com/80265277/151861065-beccaa69-5f1e-482b-a165-27e09ab2adf4.png)

when I try to connect , connect for a second and connection reset after.

**Reply by FKLCo** [reply]
![bcnc version](https://user-images.githubusercontent.com/80265277/151861443-5241be23-7ece-496f-8081-2faa4850bfc5.png)

**Reply by FKLCo** [reply]
![Captură de ecran la 2022-01-31 21-54-49](https://user-images.githubusercontent.com/80265277/151863294-e204b7b7-e2de-41f7-b2a9-6fefc8d4446c.png)
Ha... works but nothing move on screen.

**Comment by Harvie** [comment]
Seems to me that you have unsupported grbl fork OR unsupported grbl configuration (namely $10, $13 or compatibility level configuration).
See README.
